src/viking.js

```diff
@@ -1,11 +1,125 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health,
+            this.strength = strength
+    }
+
+    attack() {
+        return this.strength
+
```

Comment:
Please be mindful about the use of empty lines. Does this empty line improve the readability of the code?

Reply:
Will keep this in mind more, I tend to forget when I'm trying to solve the iterations. (Any extensions that can help me with it?)

Reply:
I am not aware of such an extension unfortunately.